packages/wonder-blocks-form/src/components/labeled-text-field.js

```diff
@@ -110,44 +132,42 @@ export default class LabeledTextField extends React.Component<Props, State> {
 
     handleValidation: (errorMessage: ?string) => mixed = (errorMessage) => {
         const {onValidation} = this.props;
-        this.setState({error: errorMessage});
-        if (onValidation) {
-            onValidation(errorMessage);
-        }
+        this.setState({error: errorMessage}, () => {
```

Comment:
Is this change related to the comments in the previous PR, right?

Reply:
Correct!